src/pybind/mgr/cephadm/module.py

```diff
@@ -1222,6 +1249,8 @@ def _run_cephadm(self,
 
         :env_vars: in format -> [KEY=VALUE, ..]
         """
+        assert isinstance(stdin, str)
```

Comment:
in line 1241:

```python
stdin: Optional[str] = "",
```

so, what is correct now? `Optional[str]` or `str`?

Reply:
This is just a trick to avoid mypy error telling that None has not "encode" method.

Reply:
then can you please update the line 1095 into

```python
                     stdin: str = "",
```

?

Reply:
thank you!